packages/lit-html/src/lit-html.ts

```diff
@@ -519,19 +527,17 @@ const getTemplateHtml = (
 };
 
 export type Template = Interface<TemplateImpl>;
+export type {TemplateImpl};
```

Comment:
Call this `TemplateType`?

Seems slightly unfortunate to export a thing called `Impl`.

Reply:
done